tests/unit/controller/PageControllerTest.php

```diff
@@ -72,12 +72,18 @@ public function testIndex() {
 		$url = 'http://owncloud/ajax/upload.php';
 		$this->mockUrlToUploadEndpoint($url);
 		$publicUploadEnabled = 'yes';
+		$mailNotificationEnabled = 'no';
+		$mailPublicNotificationEnabled = 'yes';
 		$params = [
 			'appName' => $this->appName,
 			'uploadUrl' => $url,
-			'publicUploadEnabled' => $publicUploadEnabled
+			'publicUploadEnabled' => $publicUploadEnabled,
+			'mailNotificationEnabled' => $mailNotificationEnabled,
+			'mailPublicNotificationEnabled' => $mailPublicNotificationEnabled
```

Comment:
One thing I've just noticed, for next time, the `=>` are not aligned.

This is what I mean:

```
		$params = [
			'appName'                       => $this->appName,
			'uploadUrl'                     => $url,
			'publicUploadEnabled'           => $publicUploadEnabled,
			'mailNotificationEnabled'       => $mailNotificationEnabled,
			'mailPublicNotificationEnabled' => $mailPublicNotificationEnabled
		];
```